### PR TITLE
tennant schema

### PR DESCRIPTION
**Pull request by tomassabol**
image

### CODE REVIEW

File: src/server/db/schema/tennant.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+export const tennant = pgTable("tennant", {
+  id: serial("id").primaryKey(),
+  name: varchar("name").notNull(),
+  basis: decimal("basis").notNull(),
+
+  pzsName: varchar("pzs_name").notNull(),
+  regionalAuthority: varchar("regional_authority").notNull(),
+
+  pzsID: integer("pzs_id").notNull(),
```
Comment (kir40lmao) [inline src/server/db/schema/tennant.ts:19]: Remove pzs related stuff - mpve to separate table called Tenant Healthcare Provider

File: src/server/db/schema/doctor.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { pgTable, serial, varchar } from "drizzle-orm/pg-core";
+
+export const doctor = pgTable("doctor", {
```
Comment (kir40lmao) [inline src/server/db/schema/doctor.ts:3]: Supposed to be head doctor, part of tenant files

File: src/server/db/schema/tennant.ts (hunk 1 above)
Comment (kir40lmao) [reply src/server/db/schema/tennant.ts:19]: Add HeadDoctor column

**Comment by tomassabol** [comment]
image

**Comment by tomassabol** [comment]
image

File: drizzle/0000_uneven_grim_reaper.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "tennant" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"name" varchar NOT NULL,
+	"basis" numeric NOT NULL,
+	"pzs_name" varchar NOT NULL,
+	"regional_authority" varchar NOT NULL,
+	"pzs_id" integer NOT NULL,
+	"doctor_id" integer NOT NULL,
```
Comment (kir40lmao) [inline drizzle/0000_uneven_grim_reaper.sql:13]: Maybe change to head_doctor_id?

File: drizzle/0001_open_karma.sql [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+CREATE TABLE IF NOT EXISTS "health_care_provider" (
+	"id" serial PRIMARY KEY NOT NULL,
+	"tennant_id" integer NOT NULL,
+	"name" varchar NOT NULL,
+	"address1" varchar NOT NULL,
+	"address2" varchar,
+	"city" varchar NOT NULL,
+	"zip" varchar NOT NULL
+);
+--> statement-breakpoint
```
Comment (kir40lmao) [inline drizzle/0001_open_karma.sql:17]: Will need an explanation on everything after this point

**Review by kir40lmao — APPROVED** [review]

**Pull request merged**